lib/zendesk_call.rb

```diff
@@ -8,7 +8,7 @@ def call(email)
 class GetTicketRequest
   def call(email, user_id)
     @user_id = user_id
-    @tickets = client.search(:query => "#{email}")
+    @tickets = client.search(:query => "#{email}", :reload => 'true')
```

Comment:
I seemed to be getting cached results after taking it out. I had many engagements and was still receiving the same 3 tickets.